core/trino-main/src/test/java/io/trino/security/TestAccessControlManager.java

```diff
@@ -152,7 +152,7 @@ public void testDenyCatalogAccessControl()
                 assertThatThrownBy(
                         () -> accessControlManager.checkCanSelectFromColumns(securityContext, new QualifiedObjectName(TEST_CATALOG_NAME, "schema", "table"), ImmutableSet.of("column")))
                         .isInstanceOf(AccessDeniedException.class)
-                        .hasMessage("Access Denied: Cannot select from columns [column] in table or view schema.table"));
+                        .hasMessage("Access Denied: Cannot select from table or view schema.table"));
```

Comment:
I think that was a good error message. I think needs to know to what things they do not have access.